docs/ldml/tr35-keyboards.md

```diff
@@ -308,6 +308,269 @@ For extensibility, the `<special>` element will be allowed at nearly every level
 
 See [Element special](tr35.md#special) in Part 1.
 
+## Normalization
+
+Normalization will not typically be the responsibility of the keyboard author, rather this will be managed by the keyboard implementation.
```

Comment:
This first sentence will not mean anything to lots of people. You need to explain at least a bit about it before launching into why I (the keyboard author) don't need to do it.

Reply:
A line or two, plus indication that good i18n libraries (such as ICU) provide functions to perform the normalization.